api/v1/installation_types.go

```diff
@@ -171,8 +171,8 @@ type InstallationSpec struct {
 	// ComponentResources or TyphaAffinity, then these overrides take precedence.
 	TyphaDeployment *TyphaDeployment `json:"typhaDeployment,omitempty"`
 
-	// CalicoWindowsUpgradeDaemonSet configures the calico-windows-upgrade DaemonSet.
```

Comment:
Sorry @tmjd, I missed your comment and am only seeing it now... One question: is there any impact of removing this field that I'm missing? I had thought that, since this PR (and the design) removes the windows upgrade daemonset itself (which is the only component that would use this field), it would be OK. I'll add it back, but I'm curious. Thanks!

Reply:
Yeah, I'm 50/50 on this one. Removing the field from the API is itself a breaking change since anyone who has GitOpts or other workflows in place to install a CR with this field set will upgrade and see that fail. But, it's disingenuous to accept this config if we know it's not going to be respected.

Safest is to leave it, release note that it is deprecated and does nothing, and then remove it from the code at a pre-determined time in the future - even if it's just the next release.

Reply:
Understood. I added it back, will add a comment stating it is deprecated and will be ignored (and removed in the future). Thanks!